compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.1'
+
+services:
+  redis:
+    image: redis
+    ports: ['6379:6379']
+    healthcheck:
+      test: ["CMD", "redis-cli","ping"]
+
+  postgres:
+    image: postgres:15
+    ports: ['5432:5432']
+    restart: always
+    environment:
+      POSTGRES_USER: app
+      POSTGRES_DB: app_development
+      POSTGRES_PASSWORD: secret
+    volumes:
+    - pgdata:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "app", "-d", "app_development"]
+      interval: 1s
+      retries: 5
+
+  server: &server
+    depends_on: ["redis", "postgres"]
+    restart: on-failure:3
+    build:
+      context: .
+      target: base
+    environment:
+      - REDIS_URL=redis://redis/0
+      - DATABASE_URL=postgres://app:secret@postgres/app_development
```

Comment:
I'm okay with that, for sure. It's written the way it is because I copied a lot of this from some other project and that compose was pulled from some _other_ project, and several generations ago I just gave up on the database name being related to the app.

I don't _think_ it really matters?